ctrlaltdel

```diff
@@ -3,8 +3,11 @@
 PATH=/usr/bin:/usr/sbin
 MSG="System is going down..."
 
+# We check for this file after receiving a SIGCONT to move to stage3
+install -m0100 /run/runit/stopit
```

Comment:
Is this allowed? The `install` man page seems to mandate a source as well. Why not `/dev/null` like it was in `1`?

Reply:
My mistaked :p 

Fixed!